spec/models/manual_spec.rb

```diff
@@ -20,15 +20,15 @@
   }
 
   let(:id) { "0123-4567-89ab-cdef" }
-  let(:updated_at) { double(:updated_at) }
-  let(:originally_published_at) { double(:originally_published_at) }
-  let(:use_originally_published_at_for_public_timestamp) { double(:use_originally_published_at_for_public_timestamp) }
-  let(:title) { double(:title) }
-  let(:summary) { double(:summary) }
-  let(:body) { double(:body) }
-  let(:organisation_slug) { double(:organisation_slug) }
-  let(:state) { double(:state) }
-  let(:slug) { double(:slug) }
+  let(:updated_at) { Time.parse("2001-01-01") }
+  let(:originally_published_at) { DateTime.parse("2002-02-02") }
```

Comment:
Is there a reason that `updated_at` is a `Time` and `originally_published_at` is a `DateTime`?